### PR TITLE
remove packages we created that we do not need

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

File: webboot/webboot.go [hunk 1]
```diff
@@ -234,12 +243,33 @@ func main() {
...
-		if err := mountkexec.KexecISO(bookmark[filename], tmp); err != nil {
-			log.Fatalf("Error in kexecISO:%v", err)
+		kernel, err := os.Open(k)
+
+		if err != nil {
+			log.Fatal(err)
+		}
+		var image boot.OSImage
+		if err := multiboot.Probe(kernel); err == nil {
+			image = &boot.MultibootImage{
```
Comment (hugelgupf) [inline webboot/webboot.go]: This image can't be right

Comment (rminnich) [reply webboot/webboot.go]: It's in the original; shall I just yank it? What is not right.

Comment (rminnich) [reply webboot/webboot.go]: I mean the original webboot stuff. That's almost certainly wrong. I'm inclined to removed this until we care about multiboot.

File: webboot/webboot.go [hunk 2]
```diff
@@ -234,12 +243,33 @@ func main() {
...
+		if err := image.Load(true); err != nil {
+			log.Fatalf("error failed to kexec into new kernel:%v", err)
+		}
+		if err := kexec.Reboot(); err != nil {
+			log.Fatalf("error failed to Reboot into new kernel:%v", err)
 		}
 	}
 
-	fmt.Printf("The URL requested: %v\n The file requested: %v\n The mounting point: %v\n", URL, filename, tmp)
+	fmt.Printf("URL: %v\n Distro: %v\nmount point: %v\n", b.URL, b, tmp)
```
Comment (bjakobson) [inline webboot/webboot.go]: why have each part be on separate lines? what’s wrong with having it all on one line?

File: webboot/webboot.go (hunk 1 above)
Comment (rminnich) [reply webboot/webboot.go]: I remove dit ...

File: webboot/webboot.go (hunk 2 above)
Comment (rminnich) [reply webboot/webboot.go]: it's showing one removed, one added; it's not two separate lines.

Comment (rminnich) [reply webboot/webboot.go]: oh I see what you mean now, oops. 
if you run it you can see. It's just more readable.

Comment (rminnich) [reply webboot/webboot.go]: ah, I see what you mean, now. 
We did it that way to make it more readable. If you try it you'll see.

**Pull request merged**